packages/workbox-build/package.json

```diff
@@ -27,6 +27,7 @@
     "@rollup/plugin-babel": "^5.2.0",
     "@rollup/plugin-node-resolve": "^11.2.1",
     "@rollup/plugin-replace": "^2.4.1",
+    "@rollup/plugin-terser": "^0.4.3",
```

Comment:
you should also delete the "rollup-plugin-terser" plugin as its replaced with "@rollup/plugin-terser"